src/general/OptionsMenu.cs

```diff
@@ -842,6 +845,7 @@ public void ApplySettingsToControls(Settings settings)
         vsync.ButtonPressed = settings.VSync;
         displayMode.Selected = DisplayModeToIndex(settings.DisplayMode);
         msaaResolution.Selected = MSAAResolutionToIndex(settings.MSAAResolution);
+        anisotropicFilterLevel.Selected = (int)settings.AnisotropicFilterLevel.Value;
```

Comment:
There should also be a reverse mapping from the anisotropic level back to the index, otherwise our GUI will blow up if Godot engine adds a new value to the enum.